### ISSUE TITLE
Crash report from Dynamo 2.19.3.6394

### ISSUE THREAD

**Issue by crazyforsloths** [DynamoDS/Dynamo#15490]
# Issue Description
Please fill in the following information to help us reproduce the issue:

## Dynamo version
Dynamo: `2.19.3.6394`

## Operating system
OS: `Microsoft Windows NT 10.0.22631.0`

## What did you do?
Run Dynamo script, getting excel data to place on the selected items corresponding parameters.

## What did you expect to see?
The parameters of the selected items be filled with the excel data corresponding to its identifier

## What did you see instead?
Crashed

## Stack Trace
```
(From the Dynamo crash window select 'Details' -> 'Copy' and paste here)
```

---
CLR: `4.0.30319.42000`

**Comment by avidit** [comment]
@crazyforsloths Is it possible for you share the dynamo graph?

**Comment by crazyforsloths** [comment]
Hi,
So it crashed. I had selected a total of 722 elements. So right now, I am
selecting one to make sure it runs correctly with one before adding the
more elements. This would be running on a project of 23000+ elements. So I
hope it is that i didn't have the graph set up correctly as far as the
lacing goes.

As for the graph, I have modified it since it crashed so idk if it will
help you replicate the problem. But let me get the screenshot n see if that
works for you.
